jenkins-admin/managed-config-files/custom/jjbini/customizedCredentialMappings.sandbox.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
similar to the previous comment, this file should be a symlink of the production one